readme.md

```diff
@@ -92,8 +94,10 @@ Using Ruby, instantiate an array called `fruits` that contains `apple`, `banana`
 Then, use an enumerator to print to the console the sentence "I'd like to eat a [fruit]" once for each fruit.
 
 ```rb
-# Your answer...
-
+fruits = ['apple', 'banana', 'orange']
+fruits.each do |fruit|
+  puts "I'd like to eat a [fruit]" once for each fruit.""}
```

Comment:
Close! But I'm not sure what this `once` stuff is about. [Take a look at the solution.](https://github.com/ga-dc/wdi7_quiz10/tree/solution#question-7)